`regions` property as CSV is broken

### TL;DR

The ability to provide a list of comma separated `region` does not seem to work

### Expected behavior

To deploy to multiple cloud run regions supplied by the `region` property

### Observed behavior

_No response_

### Action YAML

```yaml
- id: 'deploy'
      uses: 'google-github-actions/deploy-cloudrun@v2'
      with:
        service: 'hello-cloud-run'
        image: 'gcr.io/cloudrun/hello'
        region: us-east4,us-central1,asia-northeast1
```


### Log output

```text
##[error]google-github-actions/deploy-cloudrun failed with: failed to execute gcloud command `gcloud run deploy app --image us-east4-docker.pkg.dev/project/docker-images/busybox:latest --update-env-vars ^,^LOG_LEVEL=Error --update-labels ^,^managed-by=github-actions,commit-sha=REDACTED --format json --region us-east4,us-central1,asia-northeast1 --async`: ERROR: (gcloud.run.deploy) Invalid value for property [api_endpoint_overrides/run]: The endpoint_overrides property must be an absolute URI beginning with http:// or https:// and ending with a trailing '/'. [https://us-east4,us-central1,asia-northeast1-run.googleapis.com/] is not a valid endpoint override.
```


### Additional information

_No response_

--- a/README.md
+++ b/README.md
@@ -229,8 +229,7 @@ jobs:
 
 -   <a name="project_id"></a><a href="#user-content-project_id"><code>project_id</code></a>: _(Optional)_ ID of the Google Cloud project in which to deploy the service.
 
--   <a name="region"></a><a href="#user-content-region"><code>region</code></a>: _(Optional, default: `us-central1`)_ Regions in which the Cloud Run services are deployed. This can be a single
-    region or a comma-separated list of regions.
+-   <a name="region"></a><a href="#user-content-region"><code>region</code></a>: _(Optional, default: `us-central1`)_ Region in which the Cloud Run services are deployed.
 
 -   <a name="gcloud_version"></a><a href="#user-content-gcloud_version"><code>gcloud_version</code></a>: _(Optional)_ Version of the Cloud SDK to install. If unspecified or set to "latest",
     the latest available gcloud SDK version for the target platform will be

--- a/action.yml
+++ b/action.yml
@@ -258,8 +258,7 @@ inputs:
 
   region:
     description: |-
-      Regions in which the Cloud Run services are deployed. This can be a single
-      region or a comma-separated list of regions.
+      Region in which the Cloud Run services are deployed.
     default: 'us-central1'
     required: false
 

